src/main/java/core/basesyntax/Model/Dog.java

```diff
@@ -1,4 +1,4 @@
-package core.basesyntax.Model;
+package core.basesyntax.model;
```

Comment:
It looks like you have incorrect package name here. Generally, it is a good idea to stick with lowercase package names.